### PR TITLE
Feature/claimants details

### PR DESCRIPTION
**Pull request by Stivaros**
Create unvalidated version of the `claimant's details` page. Note that this is based on a screenshot so there may be conditional fields hidden underneath the radio buttons.

### CODE REVIEW

File: app/forms/claimants_detail.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class ClaimantsDetail < BaseForm
```
Comment (garytaylor) [inline app/forms/claimants_detail.rb]: Looks like you haven't done a test for this form ?

File: app/views/claimants_details/edit.html.slim [hunk 2]
```diff
@@ -0,0 +1,61 @@
+/ TODO: Add required fields
+/ TODO: Check if there are "hidden" elements on the Jadu form
+.content-body.claimants-details
+  - content_for :form_page_title do
+    = "Claimant's Details"
```
Comment (garytaylor) [inline app/views/claimants_details/edit.html.slim]: Only just noticed that we are not doing I18n translation - probably not checked this in previous PR's also.  Maybe do a jira ticket to do it retrospectively for all views ?  or go back and sort it now and do it as you work in future views - the choice is yours as long as we make sure its done by the end.

Comment (Stivaros) [reply app/views/claimants_details/edit.html.slim]: I've made a ticket, RST-893.

My tests are running from an external content file though, in preparation.

**Comment by Stivaros** [comment]
Closed as the form has hidden fields which need to be built.

**Pull request closed without merging**